blog-service/2023-02-06-metrics-monitors.md

```diff
@@ -0,0 +1,15 @@
+---
+title: Metrics Monitors Window Update
+image: https://www.sumologic.com/img/logo.svg
+keywords:
+  - monitors
+  - alerts
+  - recovery
+  - false positive alerts
+hide_table_of_contents: true
+authors:
+  - url: https://help.sumologic.com/release-notes-service/rss.xml
+    image_url: /img/release-notes/rss-orange.png
+---
+
+We've made improvements to our metric monitoring system to ensure more accurate alerts. You can now set a minimum number of data points required to trigger an alert, with the default being set to 2 data points to reduce false positives. This update will help provide more reliable alerts and reduce false positive alerts caused by intermittent or incomplete data. We encourage you to review your alert settings to ensure they meet your needs. [Learn more](https://help.sumologic.com/docs/alerts/monitors/create-monitor/#metrics-trigger-types).
```

Comment:
```suggestion
We've improved our metric monitoring system to ensure more accurate alerts. You can now set a minimum number of data points required to trigger an alert, with the default being set to 2 data points to reduce false positives. This update will help provide more reliable alerts and reduce false positive alerts caused by intermittent or incomplete data. We encourage you to review your alert settings to ensure they meet your needs. [Learn more](https://help.sumologic.com/docs/alerts/monitors/create-monitor/#metrics-trigger-types).
```

Reply:
@nmohantyctr: Thank you for the suggestion. Both "We have made improvements" and "We have improved" are grammatically correct, but "We have made improvements" is more commonly used and may be more clear and more concise for our intended audience in a release note context.